src/Fitness.js

```diff
@@ -0,0 +1,14 @@
+class Fitness {
```

Comment:
Thank you for creating a Fitness class following our instruction guidelines regarding the inheritance. glad we could refactor some repetitive functions